bundle/src/main/java/com/adobe/acs/commons/cmf/impl/ContentMigrationServlet.java

```diff
@@ -0,0 +1,88 @@
+package com.adobe.acs.commons.cmf.impl;
+
+import java.io.IOException;
+import java.util.Iterator;
+import java.util.Set;
+
+import javax.servlet.ServletOutputStream;
+
+import org.apache.felix.scr.annotations.Reference;
+import org.apache.felix.scr.annotations.sling.SlingServlet;
+import org.apache.sling.api.SlingHttpServletRequest;
+import org.apache.sling.api.SlingHttpServletResponse;
+import org.apache.sling.api.resource.Resource;
+import org.apache.sling.api.servlets.SlingAllMethodsServlet;
+
+import com.adobe.acs.commons.cmf.ContentMigrationProcessor;
+import com.adobe.acs.commons.cmf.IdentifiedResources;
+import com.adobe.acs.commons.cmf.NoSuchContentMigrationStepException;
+
+@SlingServlet(
+        methods = { "GET" },
+        resourceTypes = { "acs-commons/components/cmf/list" },
+        selectors = { "list" },
+        extensions = { "html" }
+)
+public class ContentMigrationServlet extends SlingAllMethodsServlet {
+	
+	private static final String PATH = "path";
+
+
+	private static final String MIGRATION_STEP = "migrationStep";
+
+
+	@Reference
+	ContentMigrationProcessor cmp;
+	
+	
+	/**
+	 * 
+	 */
+	private static final long serialVersionUID = 1L;
+
+	protected void doGet (SlingHttpServletRequest request, SlingHttpServletResponse response) throws IOException {
+		
+		ServletOutputStream out = response.getOutputStream();
+		Set<String> registeredProcessors = cmp.listAvailableMigrationSteps();
+		
+		out.print("List of registered processors (" + registeredProcessors.size() + ")");
+		Iterator<String> iter = registeredProcessors.iterator();
+		
+		out.print("<ul>");
+		while (iter.hasNext()) {
+			String s = iter.next();
+			out.println("<li>" + s + "</li>");
+		}
+		out.print("</ul>");
+		
+		String usedMigrationStep = request.getParameter(MIGRATION_STEP);
+		String path = request.getParameter(PATH);
```

Comment:
Should the params be collected in the Steps? 
I assume custom steps will require a number of other params.